app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsCardView.kt

```diff
@@ -10,11 +10,17 @@ import kotlinx.android.synthetic.main.view_suggested_edits_card.view.*
 import org.wikipedia.R
 import org.wikipedia.descriptions.DescriptionEditActivity
 import org.wikipedia.descriptions.DescriptionEditActivity.Action.*
+import org.wikipedia.feed.view.CardFooterView
 import org.wikipedia.feed.view.DefaultFeedCardView
 import org.wikipedia.feed.view.FeedAdapter
 import org.wikipedia.views.PositionAwareFragmentStateAdapter
 
-class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEditsCard>(context), SuggestedEditsFeedClient.Callback {
+class SuggestedEditsCardView(context: Context) : DefaultFeedCardView<SuggestedEditsCard>(context)
+        , SuggestedEditsFeedClient.Callback, CardFooterView.Callback {
```

Comment:
Very minor but can we have the line break after the comma instead of before the comma?

Reply:
@sharvaniharan I'll self merge the follow-up PR soon.